src/components/Input.js

```diff
@@ -0,0 +1,230 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classnames from 'classnames';
+import defaultTheme from '../themes/DefaultTheme';
+import darkTheme from '../themes/DarkTheme';
+import { ErrorIcon } from './icons';
+
+class Input extends React.PureComponent {
+  constructor(props) {
+    super(props);
+  }
+
+  render() {
+    const {
+      name,
+      className,
+      type,
+      value,
+      placeholder,
+      label,
+      sublabel,
+      info,
+      error,
+      autoFocus,
+      readOnly,
+      disabled,
+      onFocus,
+      onKeyPress,
+      onChange,
+    } = this.props;
+
+    return (
+      <div className="drui-inputContainer">
+        {label && <span className="drui-input-label">{label}</span>}
```

Comment:
Please, stick with the class naming as we use in our other projects.
E.g. this class name should be `drui-input__label` or `drui-inputLabel`.
Please, amend the rest of those class names too ;)